src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -60,8 +61,8 @@
     // abstract, this is fine (as long as no logging has to happen statically in this class).
     protected final Logger logger = LogManager.getLogger(this.getClass());
 
-    @Argument(fullName = StandardArgumentDefinitions.TMP_DIR_NAME, common=true, optional=true)
-    public List<File> TMP_DIR = new ArrayList<>();
+    @Argument(fullName = StandardArgumentDefinitions.TMP_DIR_NAME, common=true, optional=true, doc = "List of temp directories to use.")
+    public List<String> TMP_DIR = new ArrayList<>();
```

Comment:
If we're going to keep this argument in GATK, can you make the type into a `String` instead of `List<String>`, since it's not doing anything intelligent right now with the List?

Reply:
If I modify that, it changes the behavior of `MarkDuplicatesGATK` and `EstimateLibraryComplexityGATK` (I guess that they are setting the temporary directory properly)

Reply:
We can add a separate argument to those tools if they really need to take a List of temporary directories. But having the GATK-wide argument take a List and then ignore all but the last element is confusing.

Reply:
True. Done.